src/test/java/core/basesyntax/DataConverterImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.converter.DataConverter;
+import core.basesyntax.converter.DataConverterImpl;
+import core.basesyntax.model.FruitTransaction;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class DataConverterImplTest {
+    private DataConverter dataConverter;
+
+    @BeforeEach
+    void setUp() {
+        dataConverter = new DataConverterImpl();
+    }
+
+    @Test
+    void convertValidData() {
```

Comment:
fix test method names. Use structure methodName_params_expectedBehavior
```suggestion
    void convertToTransactions_validData_ok() {
```